### ISSUE TITLE
Loading Training data... Generate Data Loader with Batch Generator... killed

### ISSUE THREAD

**Issue by duyicute** [antonioguj/bronchinet#30]
The training loss is always around 1, what should I do to make it gradually decrease?
Does anyone have a similar problem?

**Comment by xl-lei** [comment]
Hi, I encountered the same problem. I would like to ask how you solved it? @duyicute @antonioguj
